shared/teams/team/nav-header/index.new.tsx

```diff
@@ -0,0 +1,266 @@
+import * as React from 'react'
+import * as Kb from '../../../common-adapters'
+import * as Styles from '../../../styles'
+import * as Container from '../../../util/container'
+import TeamMenu from '../menu-container'
+import {TeamID} from '../../../constants/types/teams'
+import {pluralize} from '../../../util/string'
+import capitalize from 'lodash/capitalize'
+
+// TODO: delete this component when we implement the invite widget
+// const _AddPeopleButton = (
+//   props: {
+//     teamID: TeamID
+//   } & Kb.OverlayParentProps
+// ) => (
+//   <>
+//     <Kb.Button
+//       label="Add members"
+//       onClick={props.toggleShowingMenu}
+//       ref={props.setAttachmentRef}
+//       small={true}
+//       type="Default"
+//       mode="Secondary"
+//     />
+//     <AddPeopleHow
+//       attachTo={props.getAttachmentRef}
+//       onHidden={props.toggleShowingMenu}
+//       teamID={props.teamID}
+//       visible={props.showingMenu}
+//     />
+//   </>
+// )
+// const AddPeopleButton = Kb.OverlayParentHOC(_AddPeopleButton)
```

Comment:
remove, maybe?

Reply:
fair enough - we have the other file for when making the new component